src/Stratis.Bitcoin.Features.Wallet/Models/RequestModels.cs

```diff
@@ -139,6 +140,15 @@ public class BuildTransactionRequest : TxFeeEstimateRequest
 
     public class SendTransactionRequest : RequestModel
     {
+        public SendTransactionRequest()
+        {
+        }
+
+        public SendTransactionRequest(string transactionHex)
```

Comment:
Is there any need for this to be added?

Reply:
Yes. Most of the SendTransactionRequest instances were created like 

`new SendTransactionRequest() {
Hex = *
}`

So this change introduces a way to create SendTransactionRequest in one line instead of 3 and make the code cleaner and more understandable.